### PR TITLE
XW-656 change smart banner test

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/SmartBannerTests.java [hunk 1]
```diff
@@ -52,43 +50,27 @@ private Colors(String hex) {
...
         banner.isSmartBannerVisible(),
         "Smart banner is closed"
     );
 
-    if ("ANDROID".equals(Configuration.getPlatform())) {
-      buttonName = BUTTON_NAME_FOR_ANDROID;
-    } else {
-      buttonName = BUTTON_NAME_FOR_IOS;
-    }
+    Assertion.assertEquals(banner.isButtonVisible() == true, true, "Button is not visible");
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/SmartBannerTests.java]: you can use 
`Assertion.assertTrue(banner.isButtonVisible(), "Button is not visible");`

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/SmartBannerTests.java [hunk 2]
```diff
@@ -52,43 +50,27 @@ private Colors(String hex) {
       {"lego", "LEGO_Wiki", Colors.YELLOW.hex},
       {"ladygaga", "Gagapedia", Colors.MAGENTA.hex}
   };
-  private static final String BUTTON_NAME_FOR_ANDROID = "Install";
-  private static final String BUTTON_NAME_FOR_IOS = "GET";
 
   @Test(groups = "MercurySmartBannerTest_001")
-  @RelatedIssue(issueID = "XW-656")
-  @InBrowser(browser = Browser.CHROME_ANDROID)
+  @InBrowser(browser = Browser.CHROME_MOBILE_MERCURY)
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/SmartBannerTests.java]: please use this instead:
```
@InBrowser(
    browser = Browser.CHROME,
    emulator = Emulator.GOOGLE_NEXUS_5
)
```

File: src/test/java/com/wikia/webdriver/common/core/geastures/MobileEmulationTouchActions.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package com.wikia.webdriver.common.core.geastures;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+
+public class MobileEmulationTouchActions {
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/common/core/geastures/MobileEmulationTouchActions.java]: I believe you can create class for executing js

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/SmartBannerTests.java (hunk 1 above)
Comment (bognix) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/SmartBannerTests.java]: ok cool - I didn't see the version with message as the second param

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/SmartBannerTests.java [hunk 4]
```diff
@@ -52,43 +50,27 @@ private Colors(String hex) {
...
   public void MercurySmartBannerTest_001_ButtonName_FixPosition_Close() {
     SmartBannerComponentObject banner = new SmartBannerComponentObject(driver);
     wikiURL = urlBuilder.getUrlForWiki(WIKIS[0][0]);
     banner.openArticleOnWikiByNameWithCbAndNoAds(wikiURL, WIKIS[0][1]);
-    PerformTouchAction touchAction = new PerformTouchAction(driver);
-
-    String buttonName;
+    JavascriptActions touchActions = new JavascriptActions(driver);
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/SmartBannerTests.java]: this is no longer `touchActions`

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**